packages/cojson/src/sync.ts

```diff
@@ -565,13 +544,18 @@ export class SyncManager {
       if (isAccountID(accountId)) {
         const account = this.local.getCoValue(accountId);
 
+        // We can't verify the transaction without the account, so we skip it and ask for a correction
         if (!account.isAvailable()) {
-          account.loadFromPeers([peer]);
-          invalidStateAssumed = true;
+          if (!coValue.correctionsRequested.has(sessionID)) {
+            coValue.correctionsRequested.add(sessionID);
+            invalidStateAssumed = true;
```

Comment:
We should time the correction request to when the account is loaded.

Going to do this in a separate PR.